languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -36035,6 +36035,17 @@ though, and has since been slightly modified)-->
     </rule>
   </category>
   <category id="CAT_TYPOGRAPHIE" name="Typographie" type="typographical">
+      <rulegroup id="PONCTUATION_SUR_ABREVIATIONS" name="Abréviations" default="temp_off">
```

Comment:
I would make the name less general, since this rule targets one very specific (and very important!) aspect of abbreviations.

Why did you make this a `rulegroup`? As long as there's only one rule, you don't need to create a rulegroup. I.e., this would work without lines 36038 and 36048.
Rules within rulegroups do not have IDs, generally.

Test your rules with the Rule Editor – the rule editor tells you, in this case, that the correction causes the same rule to match again:

Before: `<example correction="cf.">Nous avons déjà vu cette notion, <marker>cf</marker> le cours précédent.</example>`
After: `<example correction="cf.">Nous avons déjà vu cette notion, <marker>cf</marker>. le cours précédent.</example>`
And so on: `<example correction="cf.">Nous avons déjà vu cette notion, <marker>cf</marker>.. le cours précédent.</example>`

How would you avoid this?

And then, there are false positives in the Rule Editor results. How about excluding "CF" (both uppercase)?

Reply:
Also (as a routine), always run the mvn test before making a pull request. If the build doesn't succeed, you can make a pull request anyway, but give it the status of a "draft pull request".

Once the mvn test succeeds **and** you don't find any false positives in the Rule Editor, you can mark it as "ready for review".

Reply:
Creation of the rulegroup : I have anticipated many addings to this rulegroup, including for example vol. for volume, coll. for collection, etc. for Et cetera/cætera, [...]